.github/CONTRIBUTING.md

```diff
@@ -35,104 +56,163 @@ $ git commit -m 'Adds support for GraphQL subscriptions'
 $ git push -u origin feature/rest-api-support
 ```
 
-Once you have pushed the changes to your remote feature branch, [Create a pull request](https://github.com/open-draft/msw/compare) to the `msw` repository in GitHub. Undergo the process of code review, where the maintainers of the library will help you to get the changes from great to awesome, and enjoy your implementation merged to `master`.
+Once you have pushed the changes to your remote feature branch, [Create a pull request](https://github.com/open-draft/msw/compare) on GitHub. Undergo the process of code review, where the maintainers of the library will help you get the changes from good to great, and enjoy your implementation merged to `master`.
+
+> Please be respectful when requesting and going through the code review. Everyone on the team is interested in merging quality, tested code, and so should be you. It may take some time and iterations to get it right, and we will assist you throughout the process.
```

Comment:
```suggestion
> Please be respectful when requesting and going through the code review. Everyone on the team is interested in merging quality, tested code, and so should you. It may take some time and iterations to get it right, and we will assist you throughout the process.
```

Reply:
Corrected.